lib/internal/test_runner/reporter/spec.js

```diff
@@ -84,7 +84,7 @@ class SpecReporter extends Transform {
     if (hasChildren) {
       // If this test has had children - it was already reported, so slightly modify the output
       const err = data.details?.error?.failureType === 'subtestsFailed' ? '' : error;
-      return `${prefix}${indent}${color}${symbols['arrow:right']}${colors.white}${title}\n${err}`;
+      return `${prefix}${indent}${color}${symbols['arrow:right']}${colors.white}${title}${err ? '\n' : ''}${err}`;
```

Comment:
Since the ternary is based on `err` it might be worth pulling the trailing `${err}` into the ternary.